protocol/encoding.go

```diff
@@ -15,12 +15,18 @@ var b64en = base64.StdEncoding.EncodeToString
 
 func MarshalTemporaryBinding(tb *merkletree.TemporaryBinding) ([]byte, error) {
 	return json.Marshal(&struct {
-		Index     string `json:"index"`
-		Value     string `json:"value"`
-		Signature string `json:"signature"`
+		Index      string `json:"index"`
+		IndexProof string `json:"index_proof"`
```

Comment:
I don't think we need an ``IndexProof`` field in the TB if it's already contained in the auth path.